include/revng/Pipeline/Loader.h

```diff
@@ -28,23 +28,23 @@ namespace pipeline {
 struct ContainerDeclaration {
   std::string Name;
   std::string Type;
-  std::string Role = "";
+  std::string Role = {};
```

Comment:
1. I'm not sure why we need this at all. I'd just drop them. I can understand for default arguments though.
2. Why do you prefer `= {}`?

Reply:
1. I prefer explicitness of "it's fine if this one has no value", especially in struct fields to indicate which one you should leave unset. Not a strong preference though.

2. I actually prefer `= ""`, but you complained about it in one of the closed threads above, so I assumed that's what you don't like. I guess I was wrong.

Reply:
I'm not sure what I was referring to, but in general I don't like arguments with default values very much.

Other than that, I'd just avoid adding any syntax that's equivalent to the default constructor (unless maybe in something like `MetaAddress::invalid()` to be extra explicit).

If anything, I'd take all the `= ""`/`= {}` we can away, as long as there are no changes in semantics.

Reply:
Please take a look at the
```
`std::string`: use implicit default constructor
```
commit.